rosette/api.py

```diff
@@ -426,6 +427,12 @@ def call(self, parameters):
         params_to_serialize = parameters.serialize(self.api.options)
         headers = {}
         if self.user_key is not None:
+
+            if parameters["customHeaders"] is not None:
+                for h in parameters["customHeaders"]:
+                    headers[h[0]] = h[1]
```

Comment:
custom headers should always be a map of string to string and prefixed by a `X-` if not already